retroshare-gui/src/util/RsSyntaxHighlighter.cpp

```diff
@@ -32,23 +32,86 @@ void RsSyntaxHighlighter::highlightBlock(const QString &text)
 	int index = 0;
 	QStringList lines = text.split(endl);
 	foreach (const QString &line, lines) {
-		if(line.trimmed().startsWith('>')) {
-			setFormat(index, line.length(), quotationFormat);
+		if(line.trimmed().startsWith(">>>>>>>>") || line.trimmed().startsWith("> > > > > > > >")) {
```

Comment:
This code is not reliable enough: what happens if due to several editing, the message starts with "> > >>>> > >" ? Some spaces have been removed and some not.
I suggest that you use a regexp to search for the string, and test in various situations.

Reply:
This might be the reason beyond the PR not working on existing quoted text.

Reply:
The PR works perfect on MimeTExtEdit (with & without space) and on RSTextBrowser without space,
but im not sure where is the bug on RSTEXTBROWSER
possible somewhere bug in read mode? normaly it must work backward compatible
but not sure why it doesnt works there.



**MimeTextEdit**
![image](https://user-images.githubusercontent.com/9952056/71672640-e9cae300-2d76-11ea-87a1-f0bb347efd6f.png)


**RSTextBrowser**
![image](https://user-images.githubusercontent.com/9952056/71672800-4cbc7a00-2d77-11ea-8e5f-8961d990e400.png)

Reply:
> This code is not reliable enough: what happens if due to several editing, the message starts with "> > >>>> > >" ? Some spaces have been removed and some not.
> I suggest that you use a regexp to search for the string, and test in various situations.

but i think this will not happen or some one editing the text manually & adding extra spacing.
when i reply now old quoted text it formats for the new quoted text then it get be highlighted in RSTextBrowser.
It was nice when some one find the issue why in RSTextbrowser doesnt works coloring with 1 space

![image](https://user-images.githubusercontent.com/9952056/71673229-8641b500-2d78-11ea-80c9-626fe3537bc4.png)

Reply:
when use "> > > > " some time the text gets be hightlighted but not on every post
![image](https://user-images.githubusercontent.com/9952056/71675272-aa53c500-2d7d-11ea-9f1d-0f3ab8e86aa1.png)

it looks like some quoted text has only 1 line? then it will stays always green? when im right
but some quoted text have own seperated line then it gets colored .